PointLocation.java

```diff
@@ -0,0 +1,6 @@
+package JavaAutomationLab;
+
+public class PointLocation {
+    private int x = 0;
```

Comment:
how do you going to work with that vars?

Reply:
can't come up with an idea why we need 2D points in the task, seems like 1D is enough to track progress in the race. That's why this class is not used for now.